pkg/systemd/services/timer-dialog.jsx

```diff
@@ -159,153 +164,189 @@ const CreateTimerDialogBody = ({ setIsOpen }) => {
                                validated={submitted && validationFailed.command ? "error" : "default"}
                                onChange={setCommand} />
                 </FormGroup>
-                <FormGroup label={_("Run")}>
-                    <FormSelect id="boot-or-specific-time"
-                                value={delay}
-                                onChange={setDelay}
-                                aria-label={_("Run")}>
-                        <FormSelectOption value="system-boot"
-                                          label={_("After system boot")} />
-                        <FormSelectOption value="specific-time"
-                                          label={_("At specific time")} />
-                    </FormSelect>
-                    { delay == "system-boot" && <FormGroup className="service-run delay-group" label={_("After")}>
-                        <TextInput value={delayNumber}
-                                   validated={submitted && validationFailed.delayNumber ? "error" : "default"}
-                                   onChange={setDelayNumber} />
-                        <FormSelect className="delay-unit"
-                                    value={delayUnit}
-                                    onChange={setDelayUnit}
-                                    aria-label={_("After")}>
-                            <FormSelectOption value="sec" label={_("Seconds")} />
-                            <FormSelectOption value="min" label={_("Minutes")} />
-                            <FormSelectOption value="hr" label={_("Hours")} />
-                            <FormSelectOption value="weeks" label={_("Weeks")} />
-                        </FormSelect>
+                <FormGroup label={_("Trigger")} hasNoPaddingTop>
+                    <Flex>
+                        <Radio value="system-boot"
+                               name="boot-or-specific-time"
+                               onChange={() => setDelay("system-boot")}
+                               isChecked={delay == "system-boot"}
+                               label={_("After system boot")} />
+                        <Radio value="specific-time"
+                               name="boot-or-specific-time"
+                               onChange={() => setDelay("specific-time")}
+                               isChecked={delay == "specific-time"}
+                               label={_("At specific time")} />
```

Comment:
I guess leaving out an ID doesn't make the radios get hooked up properly in PatternFly, so each needs its own unique ID.

This really should be a compile time error for PF, or PF should wrap the label around the radio. Either-or, but right now, neither happens, so we need to manually ensure there's an ID.